src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { findTab } from '../../functions';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = findTab(tabs, selectedTabId);
```

Comment:
```suggestion
  const selectedTab = findTab(tabs, selectedTabId);
```

Reply:
It's a better name, because as  here we are rendering list of tabs, `currentTab` means that this is a tab on some iteration step